### PR TITLE
Added section on MVVM under Project Structure / Architecture

### PR DESCRIPTION
**Pull request by Dumblydore**
**What's this do?**
Updates the main readme to add a section about MVVM

**Why are we doing this? (w/ JIRA link if applicable)**
https://jira.nypl.org/browse/SIMPLY-3650

**How should this be tested? / Do these changes have associated tests?**
No test needed

**Dependencies for merging? Releasing to production?**
None

**Has the application documentation been updated for these changes?**
Yes

**Did someone actually run this code to verify it works?**
No code changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,6 +230,12 @@ architecture distributed over the application modules. The _controller_ in the a
 task-based and executes all tasks on a background thread to avoid any possibility of blocking
 the Android UI thread.
 
+#### MVVM
+
+Newer application modules, roughly follows an [MVVM](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel) architecture.
```
Comment (io7m) [inline README.md:235]: `s/follows/follow/`

:stuck_out_tongue:

**Comment by io7m** [comment]
The joys of overly pedantic review. :laughing:

**Review by io7m — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Dumblydore) [reply README.md]: 😂  https://youtu.be/e1ghnsE6B0k?t=7

Comment (io7m) [reply README.md]: :laughing:

**Pull request merged**